libs/sdk-ui-charts/src/highcharts/chart/highcharts/test/customConfiguration.test.ts

```diff
@@ -183,6 +183,44 @@ describe("getCustomizedConfiguration", () => {
             expect(result.xAxis[0]).toEqual(expectedResult);
         });
 
+        it("should set X axis configurations with the minRange property when the zooming is enabled", () => {
+            const result = getCustomizedConfiguration(
+                {
+                    ...chartOptions,
+                },
+                {
+                    chart: {
+                        zoomType: "x",
+                    },
+                },
+            );
+
+            const expectedResult = {
+                ...result.xAxis[0],
+                minRange: 2,
+            };
+
+            expect(result.xAxis[0]).toEqual(expectedResult);
+        });
+
+        it("should not set X axis configurations with the minRange property when the zooming is disabled", () => {
```

Comment:
these tests looks similar, wanna use `it.each` for them?

Reply:
Updated!